### PR TITLE
Add HTML, CSS and Linters 

### PR DESCRIPTION
**Pull request by karelvanoordt**

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
```
Comment (afanimartin) [inline .stylelintrc.json:6]: Please refer to [this](https://stackoverflow.com/questions/60345326/eslint-no-files-matching-the-pattern-lint-were-found-please-check-for-typing) answer on StackOverflow to fix the error: `No files matching the pattern "."`

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @karelvanoordt, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Please add a description to your PR. Tell us what the project is all about. What did you do?
- [ ] Please update your readme to make it more professional. Add instructions for cloning the repo, installing packages, running the app locally. This will make it easy for anyone to get up and running quickly with your project.
- [ ] Please fix all linter errors. Check my comment for suggestions.
- [ ] Please add the `test.md` file to `.gitignore` file. The file is meant for testing purposes only and should not be committed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @karelvanoordt, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] As previously requested, add a PR description.
> Please add a description to your PR. Tell us what the project is all about. What did you do?

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by karelvanoordt** [comment]
This is the latest update of my repo. These were for the HTML and CSS files. After that, the linters were correctly installed and passed. 
Thanks for the helpful reviews.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Karel Van Oordt, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL BUT RECOMMENDED
- [ ] PR should have a description of the features implemented.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**